package-lock.json

```diff
@@ -28,6 +28,13 @@
         "@agoric/transform-bang": "^0.3.1",
         "@babel/generator": "^7.5.5",
         "esm": "^3.2.5"
+      },
```

Comment:
what do the package-lock.json changes do?

Reply:
I thought I had reverted this. Let me try again.
The bug this test case is meant to reveal is provoked by the current draft of the eventual-send code. 

The test case is intended to succeed in the current master, but be ready to fail if the bug in the draft eventual-send code isn't fixed.